test/testOpenwhisk.js

```diff
@@ -134,6 +134,7 @@ describe('Testing OpenWhisk adapter', () => {
         path: '/test',
         extension: 'html',
         selector: 'print.preview',
+        url: '/test.print.preview.html?a=42&b=green',
```

Comment:
so...`path` is actually `resourcePath` ?

Reply:
also: add a test using a path to a md file, eg:

`/api/v1/webhtml?owner=tripodsan&repo=hlxtest&ref=master&path=/index.md&selector=&extension=html&strain=default&params=`

Reply:
actually the test case is wrong: `path` always points to a md file. I fixed the test.